src/app/sagas/rootSaga.ts

```diff
@@ -1,10 +1,16 @@
 import { all, AllEffect } from "redux-saga/effects";
+import { Sdk } from "../../Sdk";
+import { FeatureRunSaga } from "./FeatureRunSaga";
+import { FeatureSaga } from "./FeatureSaga";
 import { SettingsSaga } from "./SettingsSaga";
 import { StarsSaga } from "./StarsSaga";
 
 export default function* rootSaga(): IterableIterator<AllEffect<any>> {
+  const api = Sdk.getInstance("");
```

Comment:
todo for later (use some kind of config)

Reply:
ts-module-boilerplate already has some kind of build time variable, so can be passed through that,